packages/backend-api/src/test/integration/jsonapi/delete.spec.ts

```diff
@@ -32,16 +33,16 @@ Object.keys(models).forEach((modelName) => {
       delete serializedModel.id;
       describe('DELETE', () => {
         it(`delete a ${modelName} object`, async () => {
-          const res = await apiClient.post(`/${modelName}`).send({ data: serializedModel });
+          const res = await chai.request(server).post(`/${modelName}`).send({ data: serializedModel });
 
-          const res2 = await apiClient.del(`${res.body.data.links.self}`);
+          const res2 = await chai.request(server).del(`${res.body.data.links.self}`);
           expect(res2).to.have.status(204);
 
-          let res3;
           try {
-            res3 = await apiClient.get(`${res.body.data.links.self}`);
+            const res3 = await chai.request(server).get(`${res.body.data.links.self}`);
+            expect(res3).to.have.status(404);
           } catch (e) {
-            expect(e.status).to.equal(404);
+            console.log(e);
```

Comment:
Maybe log error?

Reply:
Sorry, debugging stuff left in place in the unit test.  I'll fix it.